Extension/src/LanguageServer/extension.ts

```diff
@@ -572,7 +572,6 @@ function onDidChangeVisibleTextEditors(editors: vscode.TextEditor[]): void {
                 if (clients.checkOwnership(client, editor.document)) {
                     if (!client.TrackedDocuments.has(editor.document)) {
                         // If not yet tracked, process as a newly opened file.  (didOpen is sent to server in client.takeOwnership()).
```

Comment:
Remove comment for deleted statement?

Reply:
The comment is still correct, and is in the context of the IF before it.  The line that was removed was redundant, as the same thing happens in takeOwnership().  However, because that happens asynchronously (after notifyWhenReady), I should put this back to ensure there isn't a race condition here that could cause us to try to open the file twice.